frontend/app/js/script.js

```diff
@@ -130,4 +130,24 @@ $(function() {
     });
   }());
 
+  $(".comment-meta a[data-action='reply']").click(function() {
+    var commentBodyDiv = $(this).closest(".comment-body");
+    var nitRaw = commentBodyDiv.data('nit-raw');
+    var nitpicker = commentBodyDiv.data('nitpicker');
+
+    // construct reply
+    var nitQuoted = "@" + nitpicker + " had commented:\n";
+    nitQuoted += nitRaw.split('\n').map(function(x) { return "> " + x; }).join('\n');
+    nitQuoted += '\n';
+
+    // switch to 'Write' tab incase 'Preview' tab was selected
+    $("tabset a:contains('Write')").trigger('click');
```

Comment:
Nice work @binoymichael.  Maybe we could change this `'Write'` selector to be internationalization friendly?